src/QueryPages/QueryPage.php

```
@@ -49,8 +49,10 @@ abstract public function getResults( $requestoptions );
 	/**
 	 * Clear the cache and save new results
 	 * @todo Implement caching for SMW query pages
+	 *
+	 * @suppress PhanTypeMissingReturnReal
 	 */
-	public function recache( $limit, $ignoreErrors = true ): void {
+	public function recache( $limit, $ignoreErrors = true ): bool {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$limit'. (undefined)

(UnusedFormalParameter)